src/common/imageio_rawspeed.cc

```diff
@@ -399,7 +398,7 @@ dt_imageio_retval_t dt_imageio_open_rawspeed_sraw(dt_image_t *img, RawImage r, d
       }
     }
   }
-  else if(cpp == 3 || cpp == 4)
+  else
```

Comment:
ok, but I would keep the comment for the readers:

   else // case cpp == 3 or 4